README.md

```diff
@@ -162,6 +164,22 @@ class AlbumAdmin(admin.ModelAdmin):
         """As above..."""
 ```
 
+Remote Fields
+-------------
+
+In addition to filtering by fields on a model, you can filter by fields
+on another model linked to first via a chain of foreign key, many-to-many,
+or one-to-one relationships - or their reverses. This should work
+automatically when specifying such fields as the `field_name`, for example
+`field_name = city__country`. Note that the last field needs to be a
+relation field.
+
+Multi-Select
+------------
+
+It is possible to filter by the `OR` combination of multiple field values.
+Just set `multiselect = True` in the `AutocompleteFilter` subclass or
```

Comment:
Typo, it should be `multi_select = True`.

Reply:
multi_select looks strange. maybe select_multiple?